### PR TITLE
doc: espressif: add parameter details for esptool and imgtool

### PR DESCRIPTION
**Pull request by sylvioalves**
esptool and imgtool require additional information that is not clear
to end user.

It also update ESP32 status for Zephyr porting a table text alignment

Signed-off-by: Sylvio Alves <sylvio.alves@espressif.com>

### CODE REVIEW

**Comment by sylvioalves** [comment]
@gustavonihei @d3zd3z PTAL

File: docs/readme-espressif.md [hunk 1]
```diff
@@ -70,20 +70,47 @@ esptool.py --chip <TARGET> elf2image --flash_mode dio --flash_freq 40m --flash_s
 ```
 esptool.py -p <PORT> -b <BAUD> --before default_reset --after hard_reset --chip <TARGET> write_flash --flash_mode dio --flash_size <FLASH_SIZE> --flash_freq 40m <BOOTLOADER_FLASH_OFFSET> build/mcuboot_<TARGET>.bin
 ```
+---
+***Note***  
+You may adjust the port `<PORT>` (like `/dev/ttyUSB0`) and baud rate `<BAUD>` (like `2000000`) according to the connection with your board.  
+You can also skip `<PORT>` and `<BAUD>` parameters so that esptool tries to automatically detect it.
+
+
```
Comment (gustavonihei) [inline docs/readme-espressif.md:78]:
```suggestion
```
Single blank line should be enough.

**Comment by gustavonihei** [comment]
@sylvioalves could you please update the PR with information about the recently supported ESP32-S3?

**Comment by sylvioalves** [comment]
@gustavonihei @almir-okato PTAL.

File: docs/readme-espressif.md [hunk 2]
```diff
@@ -8,10 +8,10 @@ Documentation about the MCUboot bootloader design, operation and features can be
...
 The current port is available for use in the following SoCs within the OSes:
 
-| | ESP32 | ESP32-S2 | ESP32-C3 |
-| ----- | ----- | ----- | ----- |
-| Zephyr | Supported | WIP | WIP |
-| NuttX | Supported | Supported | Supported |
+| | ESP32 | ESP32-S2 | ESP32-C3 | ESP32-S3 |
+| :-----: | :-----: | :-----: | :-----: | :-----: |
+| Zephyr | Supported | Supported | Supported | WIP |
+| NuttX | Supported | Supported | Supported | Supported |
```
Comment (almir-okato) [inline docs/readme-espressif.md:14]: I guess that ESP32-S3 is currently WIP on NuttX for use with MCUboot

**Comment by almir-okato** [comment]
Line 359 you could add ESP32-S3 as well. (I don't know why I can't comment directly on that line)

Overall LGTM

**Review by d3zd3z — APPROVED** [review]

**Review by gustavonihei — APPROVED** [review]

**Pull request merged**